### PR TITLE
Init docs for Big Query Transformations

### PR DESCRIPTION
**Pull request by AdamVyborny**
JIRA: https://keboola.atlassian.net/browse/PST-1153

### CODE REVIEW

**Review by pavel-synek — APPROVED** [review]
good in my perspective, but please check with Jirka Semmler as well.

**Comment by AdamVyborny** [comment]
@jirkasemmler můžeš na to pls někdy mrknout za engg?

File: transformations/bigquery/index.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+
+[BigQuery](https://cloud.google.com/bigquery) offers a range of features:
+
+- Fully managed, serverless data warehouse
+- Automatic scaling of compute resources
+- Storage and analysis of multi-terabyte datasets
+- High-speed streaming insertion of data
+- Integrates with Google's data analytics ecosystem
+
+## Limits
```
Comment (jirkasemmler) [inline transformations/bigquery/index.md:17]: tady jsem to ted popisoval, jake to ma limity jeste ze strany kebooly https://help.keboola.com/storage/byodb/#bigquery-limitations

**Review by jirkasemmler — CHANGES_REQUESTED** [review]
jen tam prosim nekam zalinkuj ty nase limitace

**Review by jirkasemmler — APPROVED** [review]

**Pull request merged**